custom_components/rainforest_emu_2/config_flow.py

```diff
@@ -156,16 +157,5 @@ async def async_get_device_properties(self, device_path, host, port) -> dict[str
                 CONF_PORT: port
             }
         _LOGGER.debug("get_devices_properties DeviceInfo response is None")
-
```

Comment:
How come remove this? I think this resolved an issue with another user where device info didnt appear to be responding.